pom.xml

```diff
@@ -23,8 +23,8 @@
   <version>1.1-SNAPSHOT</version>
   <packaging>hpi</packaging>
 
-  <name>Service Fabric Plugin</name>
-  <description>Plugin for deploying service fabric applications.</description>
+  <name>Azure Service Fabric Plugin</name>
+  <description>Plugin for deploying service fabric applications on azure or on premises or other public clouds.</description>
```

Comment:
Lets just change the description to Plugin for deploying Azure Service Fabric applications.